src/db/collections/analytics/influx.rs

```diff
@@ -58,18 +58,10 @@ impl From<Measurement> for influxdb::WriteQuery {
                 ),
             Measurement::LedgerSizeAnalytics(m) => m
                 .prepare_query("stardust_ledger_size")
-                .add_field(
-                    "total_storage_deposit_value",
-                    m.inner.total_storage_deposit_value.to_string().parse::<u64>().unwrap(),
-                )
-                .add_field(
-                    "total_key_bytes",
-                    m.inner.total_key_bytes.to_string().parse::<u64>().unwrap(),
-                )
-                .add_field(
-                    "total_data_bytes",
-                    m.inner.total_data_bytes.to_string().parse::<u64>().unwrap(),
-                ),
+                .add_field("total_storage_deposit_value", m.inner.total_storage_deposit_value)
+                .add_field("total_key_bytes", m.inner.total_key_bytes)
+                .add_field("total_data_bytes", m.inner.total_data_bytes)
+                .add_field("total_byte_cost", m.inner.total_byte_cost),
```

Comment:
Rename this to `total_storage_deposit_value` and remove the previous statistic based on the unlock condition.

Reply:
Like, remove it from everywhere?

Reply:
Yes and replace it with `m.inner.total_byte_cost` please.